docker/api

```diff
@@ -0,0 +1,11 @@
+FROM ruby
+MAINTAINER Jonathan Claudius
+ENV PROJECT=github.com/mozilla/ssh_scan_api
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
+RUN gem install bundler
+RUN bundle install
```

Comment:
protip: each RUN creates a new docker layer which makes the image larger. Bundle commands into a single RUN block to avoid it.